modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -0,0 +1,111 @@
+[id="cluster-logging-configuration-of-json-log-data-for-default-elasticsearch_{context}"]
+= Configuration of JSON log data for default Elasticsearch
```

Comment:
Should be gerund "Configuring JSON log data for default Elasticsearch"

Reply:
Updated. Thanks.